src/utils.ts

```diff
@@ -173,6 +173,19 @@ export async function executeScript(
   return terminal
 }
 
+export async function stopJavaProcess() {
```

Comment:
Is there no decent os-lib type library for typescript/javascript?

Reply:
No there is not, I am not to sure how common it is to run scripts on the OS through TS/JS like we are doing. But in another PR I have coming it up it split out a lot the OS specific stuff into separate functions to allow for better use of those items. But yeah I couldn't find any package that would be what we want, the closest thing is `run-script-os` but that is for the `package.json` scripts to make OS specific commands that way but nothing like what we would need.